server/td-agent.conf

```diff
@@ -0,0 +1,48 @@
+<system>
+  log_level debug
+</system>
+
+## match tag=debug.** and dump to console
+<match debug.**>
+  @type stdout
+  @id output_stdout
+</match>
+
+####
+## Source descriptions:
+##
+
+## built-in TCP input
+## @see http://docs.fluentd.org/articles/in_forward
+<source>
+  @type forward
+  <transport tls>
+    cert_path /etc/td-agent/certs/fluentd.crt
+    private_key_path /etc/td-agent/certs/fluentd.key
+    private_key_passphrase PASSPHRASE_HERE
```

Comment:
Can we use an environment variable here instead of having to change the config file on deployment? I found some info on the fluentd site: https://docs.fluentd.org/v0.12/articles/faq#how-can-i-use-environment-variables-to-configure-parameters-dynamically

Reply:
It is possible, yes.

However, the current setup starts the daemon using systemd (since it is on Ubuntu 18.04).  So making a change to the environment would then mean changing `/opt/td-agent/etc/systemd/td-agent.service`.  I don't know if moving this from one config file to another is what you had in mind.

If we use environment variables, what mechanism are you proposing for starting up the daemon?